pkgs/by-name/di/digiham/package.nix

```
@@ -31,11 +37,15 @@ stdenv.mkDerivation rec {
     icu
   ];
 
-  meta = with lib; {
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgram = "${builtins.placeholder "out"}/bin/dmr_decoder";
+  doInstallCheck = true;
+
+  meta = {
     homepage = "https://github.com/jketterl/digiham";
     description = "tools for decoding digital ham communication";
```

```suggestion
    description = "Tools for decoding digital ham communication";
```
(might as well fix more style issues)

---

Fixed, thanks